### DISCUSSION TITLE
Netflix Quality Check - Suggested Fix Column

### DISCUSSION THREAD

**Discussion by tharsim** [SubtitleEdit/subtitleedit#6911] [Q&A]
After running the Netflix Quality Check in SE, I get a list of suggested fixes to adjust my file according to Netflix's guidelines. Since there isn't an "apply fixes" button, I have to make these adjustments manually.

Is there any way to copy the "Suggested Fix" column or access it separately, so I can attempt to automate the fixes using an independent solution? I'm aware that SE generates a CSV report, but it only provides a summary without the specific fixes.

**Comment by matmaggi** [comment]
I think @OmrSi can help with that

**Comment by OmrSi** [comment]
@niksedk Would it be okay to add the fix column to the CSV as well?

**Comment by Gnefitisis-3** [comment]
Any updates on this issue?

**Comment by OmrSi** [comment]
I never noticed the thumbs-up. :3

**Comment by OmrSi** [comment]
https://github.com/SubtitleEdit/subtitleedit/pull/10232
